.github/linters/.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 115
 # http://pep8.readthedocs.org/en/latest/intro.html#error-codes
-ignore = E128, E265, F403, W503, W1618
```

Comment:
Not sure what W1618 is, can't find documentation on it.

Reply:
these come from pylint:
https://pylint.readthedocs.io/en/1.9/technical_reference/features.html
> no-absolute-import (W1618):
>   | import missing `from __future__ import absolute_import`
> Used when an import is not accompanied by from __future__ import absolute_import (default behaviour in Python 3)

That particular error is a holdover from the migration from Python 2 to 3